lib/foreman_scap_client/base_client.rb

```diff
@@ -81,7 +81,7 @@ def scan
     def run_scan
       stdout_str, error_str, result = Open3.capture3(scan_command_env_vars, scan_command)
       if result.success? || result.exitstatus == 2
-        puts error_str.split("\n").select { |item| item.start_with?('WARNING:') || item.start_with?('Downloading') }.join("\n")
+        puts error_str.scrub('').split("\n").select { |item| item.start_with?('WARNING:') || item.start_with?('Downloading') }.join("\n")
```

Comment:
Should we at least put in question marks or something instead of the parts which are encoded in non-utf8?

Reply:
I guess that makes sense, updated.